client/src/templates/Challenges/components/interactive-editor.css

```diff
@@ -23,7 +23,7 @@
 }
 
 .sp-preview-actions .sp-button:hover {
-  background-color: var(--gray-10);
+  background-color: var(--gray-10) !important;
   color: var(--gray-90) !important;
-  border-color: var(--gray-90);
+  border-color: var(--gray-90) !important;
```

Comment:
Also a drive-by change.

These styles were added in #62848 to address an a11y issue. But apparently they are not effective without `!important`.